addons/purchase/static/src/js/product_discount_field.js

```diff
@@ -0,0 +1,68 @@
+/** @odoo-module **/
```

Comment:
move the sale field in product instead of duplicating everything.

Reply:
Requested changes are done, can you please check ?

Reply:
@amoyaux